### ISSUE TITLE
Records format broken when there is a comment in the middle

### ISSUE THREAD

**Issue by DavidHernandez** [avh4/elm-format#257]
Comes from https://github.com/avh4/elm-format/issues/251

I have a complex record and I want to document it. This is how I expected it to look:

```
type alias CustomRecord =
     -- Comment
    { field : String
    , field2 : String
    -- Comment
    , field3 : String
    , field4 : String
  }
```

And this is how it looks:

```
type alias CustomRecord =
     -- Comment
    { field : String
    , field2 :
        String
        -- Comment
    , field3 : String
    , field4 : String
  }
```

**Comment by avh4** [comment]
(This is similar to https://github.com/avh4/elm-format/issues/133)

**Comment by avh4** [comment]
This is now a duplicate of https://github.com/avh4/elm-format/issues/162

**Comment by avh4** [comment]
Fixed for elm-format 0.6.0-alpha
